src/features/content-answers/ContentAnswersModal.scss

```diff
@@ -46,4 +46,10 @@
     .modal-close-button {
         top: 20px;
     }
+
+    .bdl-ContentAnswersModalFooter-submitButton {
```

Comment:
had to move this up because it was getting overrwritten by the original button classes